enterprise-suite/templates/alertmanager-pvc.yaml

```diff
@@ -1,12 +1,12 @@
-{{ if (and .Values.createAlertManager .Values.managePersistentVolumes) }}
+{{ if (and .Values.createAlertManager .Values.usePersistentVolumes) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: alertmanager-storage
-  annotations:
-    "helm.sh/resource-policy": keep
```

Comment:
This makes me pretty uneasy still - this is our only failsafe.

Reply:
Not 100% sure on this one.  Keeping the PVC's around is what made things confusing for us developers.  I think we just need to ensure that `lbc.py` makes the choices clear to normal users.  If they specify `--force-install` then that's on them...  Or not...